inflate.h

```diff
@@ -121,7 +121,7 @@ struct ALIGNED_(64) inflate_state {
     uint32_t chunksize;         /* size of memory copying chunk */
 
         /* bit accumulator */
-    uint32_t hold;              /* input bit accumulator */
+    uint64_t hold;              /* input bit accumulator */
```

Comment:
Below is the struct member defined:
```c
uint32_t padding[2];
```

Maybe we should decrease it to 1?

Reply:
Will test...